indico/modules/rb_new/util.py

```diff
@@ -151,6 +151,10 @@ def get_existing_room_occurrences(room, start_dt, end_dt, allow_overlapping=Fals
         query = query.filter(db_dates_overlap(ReservationOccurrence, 'start_dt', start_dt, 'end_dt', end_dt))
     else:
         query = query.filter(ReservationOccurrence.start_dt >= start_dt, ReservationOccurrence.end_dt <= end_dt)
+    if only_accepted:
+        query = query.filter(Reservation.is_accepted)
+    if asc:
```

Comment:
Why don't we just always sort it? Currently the order is arbitrary when not using `asc=True`.